Assets/Scripts/SimManager.cs

```diff
@@ -15,26 +15,35 @@ public class SimManager : MonoBehaviour {
     { Configs.AgentType.MissileInterceptor, "MissileInterceptor" },
     { Configs.AgentType.FixedWingThreat, "FixedWingThreat" },
     { Configs.AgentType.RotaryWingThreat, "RotaryWingThreat" },
-
   };
 
-  public Configs.SimulatorConfig simulatorConfig;
+  // Map from the attack behavior type to the attack behavior configuration file.
+  // TODO(titan): Replace the JSON files with Protobuf text format.
+  private static readonly Dictionary<Configs.AttackBehaviorType, string> AttackBehaviorMap = new() {
+    { Configs.AttackBehaviorType.DefaultDirectAttack, "default_direct_attack.json" },
+    { Configs.AttackBehaviorType.BrahmosDirectAttack, "brahmos_direct_attack.json" },
+    { Configs.AttackBehaviorType.Fateh110BDirectAttack, "fateh110b_direct_attack.json" },
+  };
 
```

Comment:
This concerns me - how do we define new AttackBehaviorTypes at runtime?

Reply:
https://github.com/PisterLab/micromissiles-unity/pull/61/files#diff-3827e524f3d9bab8ab0d32c9b0d9e4eeb567bc8212fe5b1034978f1d8de5ba08R45-R63

The Protobuf supports `oneof`, where either one of the fields can be set.  For the default configs, there are enums to make the configurations a bit cleaner.  For custom external configs, the `.pbtxt` file name can be directly specified for the configuration.  Similarly, the attack behavior can be one of the default behaviors or can be a `.pbtxt` file name.